staging/src/k8s.io/client-go/tools/cache/shared_informer.go

```diff
@@ -575,6 +581,14 @@ func (s *sharedIndexInformer) AddEventHandlerWithResyncPeriod(handler ResourceEv
 	}
 }
 
+func (s *sharedIndexInformer) RemoveEventHandler(handler ResourceEventHandler) bool {
```

Comment:
i don't see it used in RQ or GC, is it going to be used?

Reply:
I will address this in the separate PR (#98536) but the short answer is that controller-runtime uses the shared informer directly (rather than through a factory) and will use this functionality.